extensions/assessmentExtension/context.json

```diff
@@ -4,12 +4,20 @@
     "extensions": "https://w3id.org/openbadges/extensions#Assessment",
     "schema": "http://schema.org/",
     "overallDescription": "extensions:overallDescription",
-    "assessment": "extensions:assessment"
+    "assessmentDescription": "extensions:assessmentDescription",
```

Comment:
Consider this context snippet:
```jsonld
{
  "@context": {
    "obi": "https://w3id.org/openbadges#",
    "extensions": "https://w3id.org/openbadges/extensions#Assessment",
    "overallDescription": "extensions:overallDescription",
  }
}
```

This maps the term `overallDescription` to `https://w3id.org/openbadges/extensions#AssessmentoverallDescription`. I don't see a matching anchor tag in the extensions/index.html file that this IRI would direct to. Is this IRI correct? 

I'm not sure the context with its remapping of "extensions" is doing what you want it to.

Ways to resolve:
1) Don't remap "extensions" and add an anchor/id for "overallDescription" to the extensions/index.md file
2) Proceed with remapping "extensions" within this context and add the anchor/id "AssessmentoverallDescription" to the extensions/index.md file